code/datums/status_effects/debuffs.dm

```diff
@@ -69,17 +69,21 @@
 /datum/status_effect/incapacitating/sleeping/tick()
 	if(owner.maxHealth)
 		var/health_ratio = owner.health / owner.maxHealth
-		if(health_ratio > 0.8)
-			var/healing = -0.2
-			if((locate(/obj/structure/bed) in owner.loc))
-				healing -= 0.3
-			else
-				if((locate(/obj/structure/table) in owner.loc))
-					healing -= 0.1
+		var/healing = -0.2
+		if((locate(/obj/structure/bed) in owner.loc))
+			healing -= 0.3
+		else if((locate(/obj/structure/table) in owner.loc))
+			healing -= 0.1
+		for(var/obj/item/bedsheet/bedsheet in range(owner.loc,0))
```

Comment:
why ``range(owner.loc,0)``? ``owner.loc`` is *basically* the same, at least in this context.

Reply:
Wouldn't some strange shenanigans happen then if someone sleeps in a locker if we do it that way?

Reply:
if you want to spend years sleeping in a locker to heal 20 damage be my guest